data/sql/updates/pending_db_world/rev_1507983586199534500.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1507983586199534500');
+
+-- Warsong Recruitment Officer - Gossip (Hellscream's Vigil)
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=19 AND (`SourceEntry`=11586 OR `SourceEntry`=11585);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceEntry`,`ConditionTypeOrReference`,`ConditionValue1`,`Comment`) VALUES 
+(19, 11585, 14, 10172, "Accept quest 11585 - Quest 10172 needs to be incomplete"),
+(19, 11586, 8, 10172, "Accept quest 11586 - Quest 10172 needs to be rewarded");
+
+UPDATE `quest_template_addon` SET `ExclusiveGroup`=11585 WHERE  `ID`IN(11585,11586);
```

Comment:
there is one extra space bertween `WHERE` and `ID`, and no space between `ID` and IN.

Please change with:

```UPDATE `quest_template_addon` SET `ExclusiveGroup`=11585 WHERE `ID` IN (11585,11586);```

so we avoid any import errors